style.css

```diff
@@ -385,3 +390,209 @@ hr {
   border-radius: 2px;
   outline: none;
 }
+
+.navbar-list {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .body-section {
+    width: 100%;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .parent {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .navbor {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-self: center;
+    background-color: #fff;
+    padding-left: 10rem;
+    padding-right: 10rem;
+  }
```

Comment:
- [ ] As requested by the previous reviewer please make sure your navbar looks just like that in the[ figma design template ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=sbVVw9cdSSnpmkR2-0)as our aim is to make an exact replica of the figma design. (hint: You have misspelled the name of the `navbar` class in the css.  It should be `navbar` instead of `navbor`) 
- [ ] Please also make sure to change the color of the logo to `#6070FF` from `#a3acff` to match the colors of the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=sbVVw9cdSSnpmkR2-0).

Your design:

<img width="1145" alt="Screenshot 2023-04-07 at 11 46 23 AM" src="https://user-images.githubusercontent.com/44798044/230559625-af026983-ae54-4d6a-bacc-f99dbc14ac46.png">

Figma design:

<img width="722" alt="Screenshot 2023-04-07 at 12 06 15 PM" src="https://user-images.githubusercontent.com/44798044/230559740-c05936de-4e95-494d-852e-a34eaa861c94.png">